[14.0] [MIG] website_sale_stock_list_preview: Migration to 14.0 (#640)
OCA/e-commerce: website_sale_stock_list_preview/readme/USAGE.rst

To use this module, you need to:

> * Edit some products on the option Availability of e-commerce

Reviewer: ```suggestion
* Edit the 'Availability' option under 'eCommerce' tab in your selected product.
```

> * Go to the shop on the website and active Add to Cart view on Customize menu.

Reviewer: ```suggestion
* Go to the website's shop and activate the 'Add to Cart' view under 'Customize' menu.
```

> * You can see on your products the results of the options that you had choose.

Reviewer: ```suggestion

You can see, on your products, the options you have chosen.
```
